### PR TITLE
docs: Firebase index creation 

### PR DESCRIPTION
**Pull request by adryan30**
This PR aims to update the docs to explain how to create a Firestore index, as mentioned as necessary in #6.

It also updates the Chatroom.tsx file to properly log on the console when firebase throws an error, this facilitates debugging and helps create the index, if necessary.

### CODE REVIEW

File: src/components/chatroom/Chatroom.tsx [hunk 1]
```diff
@@ -25,11 +25,12 @@ export const Chatroom = (props: {
     .orderBy('createdAt')
     .limit(25)
     .where("threadId", "==", id)
-  const [messages]: [
+  const [messages,, err]: [
     IMessage[] | undefined,
     boolean,
     firebase.FirebaseError | undefined
   ] = useCollectionData(query, { idField: 'id' })
+  if (err) console.error(err)
```
Comment (benji011) [inline src/components/chatroom/Chatroom.tsx]: for development purposes this is fine but, i would prefer `err` & logging to console removed from production

File: docs/FIREBASE_SETTINGS.md [hunk 2]
```diff
@@ -24,6 +24,24 @@
...
+4. Choose these settings:
+   1. For the collection ID: messages
+   2. For the fields to index:
+      1. threadId: Ascending
+      2. createdAt: Ascending
+   3. Choose the "Collection" query scope
+5. Click `Create index` and wait for it to build
+
+An alternative method to build this index, is entering any chatroom and clicking the firebase error that might appear in the console. (This method is not recommeded)
+
```
Comment (benji011) [inline docs/FIREBASE_SETTINGS.md]: This probably isn't needed & can be removed

```suggestion
```

**Review by benji011 — APPROVED** [review]
Looking good! I left a couple comments which are minor changes. But otherwise once you've pushed the changes they can be merged :)

**Comment by adryan30** [comment]
Just applied the necessary changes! Thanks for the feedback.

**Comment by benji011** [comment]
Thanks @adryan30 !

**Pull request merged**